hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestStorageRestore.java

```diff
@@ -294,7 +294,7 @@ public void testDfsAdminCmd() throws Exception {
       restore = fsi.getStorage().getRestoreFailedStorage();
       assertTrue("After check call restore is " + restore, restore);
       String commandOutput = cmdResult.getCommandOutput();
-      commandOutput.trim();
+      commandOutput = commandOutput.trim();
       assertTrue(commandOutput.contains("restoreFailedStorage is set to true"));
```

Comment:
I think we can remove ``commandOutput.trim();`` itself. The ``assertTrue`` is anyway checking ``contains`` which will pass irrespective of this ``trim``

Reply:
Indeed. Now I realize that deleting my forked repo rather hastily was a mistake :( I've tried various ways to edit this PR branch but didn't succeed. It seems that it leaves us with two options: either I close this PR and open a new one, or someone with the right permissions edit it. Let me know what you think is best. Sorry for the inconvenience.

Reply:
See, if you can restore the repository
https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository

Reply:
It says I have to ask GitHub support to restore a forked repo. I've created a ticket to request that, let's see if they can help. Thanks

Reply:
Your repo seems back
https://github.com/thomasleplus/hadoop/tree/patch-1

Reply:
Yes, they are quick!

Reply:
I pushed the change as requested. Thanks for your patience.